src/gui/resources/main_window.qml

```diff
@@ -101,6 +101,56 @@ Window {
         }
 
     }
+    Item {
```

Comment:
could you make this a seperate file like the one that I did for the drive mode. Makes it a lot more modular/reusable,